commands/operator-sdk/cmd/add/crd.go

```diff
@@ -59,49 +56,64 @@ Generated CR  filename: <project-name>/deploy/<group>_<version>_<kind>_cr.yaml
 }
 
 func crdFunc(cmd *cobra.Command, args []string) {
+	cfg := &input.Config{
+		Repo:           cmdutil.MustGetwd(),
```

Comment:
@shawn-hurley @estroz 
I think Config.Repo is supposed to be the go import for the project and not the absolute path.
```Go
			cfg := &input.Config{
				Repo:           cmdutil.CheckAndGetCurrPkg(),
				AbsProjectPath: cmdutil.MustGetwd(),
			}
```
https://github.com/operator-framework/operator-sdk/blob/refactor/rebase-master-add-AO-commands/pkg/scaffold/input/input.go#L113-L114

Since the `Repo` field isn't really used in the CRD template this doesn't cause an issue but it really should be the project path relative to`$GOPATH/src`.

Reply:
Yes that is correct and should be changed to match your example.

Reply:
So an ansible user cannot create new CRDs unless they set up their project with a gopath? I don't think this is a change we want.

Reply:
The `Repo` field isn't used by CRD or CR scaffolds, so it either shouldn't be set at all or should be set to the Go package location.

Reply:
No they still should be able to set it up.
`Repo ` `AbsProjectPath ` `ProjectName` are just fields common to all inputs.

For generating a CRD you only need `AbsProjectPath` which is set correctly. 
`Repo` is mostly used for Go files templates to set the go imports correctly.

You can just ignore `Repo` not set it.
```Go
			cfg := &input.Config{
				AbsProjectPath: cmdutil.MustGetwd(),
			}
```

Reply:
> The Repo field isn't used by CRD or CR scaffolds, so it either shouldn't be set at all or should be set to the Go package location.

We should probably rename this then for the future. `Repo` sounds like it is just the base repository directory, and I would assume that we would be able to use it in the future if we needed it. If you really think it should only be the  go package location then we may want to create a new field for that. As @hasbro17 said though we can just remove them right now from our new scaffolds and we can come back to this.